% Latex template: mahmoud.s.fahmy@students.kasralainy.edu.eg
% For more details: https://www.sharelatex.com/learn/Beamer

\documentclass[aspectratio=1610]{beamer}					% Document class

\setbeamertemplate{footline}[text line]{%
  \parbox{\linewidth}{\vspace*{-8pt}A brief introduction to graphical models and deep methods in computational network biology \hfill\insertshortauthor\hfill\insertpagenumber}}
\setbeamertemplate{navigation symbols}{}

\usepackage[english]{babel}				% Set language
\usepackage[utf8x]{inputenc}			% Set encoding

\mode<presentation>						% Set options
{
  \usetheme{default}					% Set theme
  \usecolortheme{default} 				% Set colors
  \usefonttheme{default}  				% Set font theme
  \setbeamertemplate{caption}[numbered]	% Set caption to be numbered
}

% Uncomment this to have the outline at the beginning of each section highlighted.
%\AtBeginSection[]
%{
%  \begin{frame}{Outline}
%    \tableofcontents[currentsection]
%  \end{frame}
%}

\usepackage{graphicx}					% For including figures
\usepackage{booktabs}					% For table rules
\usepackage{hyperref}					% For cross-referencing
\usepackage[absolute,overlay]{textpos}
\usepackage{bm}
\usepackage[font=small,labelfont=bf]{caption}

\title{Dynamics on gene networks}	% Presentation title
\author{Clayton W. Seitz}								% Presentation author
\date{\today}									% Today's date	

\begin{document}

% Title page
% This page includes the informations defined earlier including title, author/s, affiliation/s and the date
\begin{frame}
  \titlepage
\end{frame}

% The following is the most frequently used slide types in beamer
% The slide structure is as follows:
%
%\begin{frame}{<slide-title>}
%	<content>
%\end{frame}

\begin{frame}{Data Summary}


\end{frame}

\begin{frame}{Research Strategy}

\begin{textblock*}{14cm}(0.75cm,0.75cm)
\begin{figure}
\captionsetup{font={footnotesize,it}}
\includegraphics[width=14cm]{logic.png}
\caption{\textbf{A} 10-gene network sampled from \emph{Saccharomyces Genome Database} (SGD). \textbf{B} Steady state values from Monte Carlo simulation. \textbf{C} Barcoding scheme for multiplexed RNA imaging (Alessio, 2021) \textbf{D} Cartoon of a joint distribution in higher dimensions \textbf{E} Relationships between components}
\end{figure}
\end{textblock*}

\end{frame}

\begin{frame}{Simulating gene expression \emph{in-silico}}
\begin{textblock*}{15cm}(0.5cm,1cm)
\begin{figure}
\includegraphics[width=10cm]{sergio.png}
\caption{(Dibaeinia and Sinha, Cell Systems 2020)}
\end{figure}
\end{textblock*}
\end{frame}

\begin{frame}{Simulating gene expression \emph{in-silico}}

\begin{textblock*}{15cm}(0.5cm,1.5cm)
We can simulate the expression of a gene as a function of the levels of its regulators (TFs), as prescribed by a
fixed gene regulatory network (GRN) 
\end{textblock*}

\begin{textblock*}{15cm}(0.5cm,2.75cm)
This is done via the chemical Langevin equation (Gillespie, 2000; Dibaeinia 2020):
\end{textblock*}

\begin{textblock*}{7cm}(4cm,3cm)
\begin{figure}
\begin{align*}
\dot{x_{i}} &= P_{i}(t) -\lambda_{i}x_{i}(t) + q_{i}\left(\sqrt{P_{i}(t)}\alpha + \sqrt{\lambda_{i}x_{i}}\beta\right)  \\
\end{align*}
\end{figure}
\end{textblock*}

\begin{textblock*}{7cm}(0.5cm,5cm)
The transcription rate of gene $i$\\
depends non-linearly on its regulators:
\begin{align*}
P_{i} &= \sum_{j}p_{ij} + b_{i}\\
p_{ij} &= m_{ij}\frac{x_{ij}^{n_{ij}}}{x_{ij}^{n_{ij}}+h_{ij}^{n_{ij}}}
\end{align*}
\end{textblock*}

\begin{textblock*}{8cm}(7cm,5cm)
\begin{figure}
\includegraphics[width=8cm]{logic-ab.png}
\end{figure}
\end{textblock*}

\end{frame}

\begin{frame}{Generating ``cell types" for differentiation analyses}

\begin{textblock*}{10cm}(0.5cm,1.5cm)
Genes A and B are so-called \emph{master regulators}
\end{textblock*}

\begin{textblock*}{10cm}(0.5cm,2.2cm)
\begin{align*}
\dot{x_{A}} &= P_{A} -\lambda_{A}x_{A} + \phi(x_{A})\\
\dot{x_{B}} &= P_{B} -\lambda_{B}x_{B} + \phi(x_{B})\\
\dot{x_{C}} &= P_{C}(A,B) -\lambda_{C}x_{C} + \phi(x_{A},x_{B},x_{C})\\
\dot{x_{D}} &= P_{D}(C) -\lambda_{D}x_{D} + \phi(x_{C},x_{D})\\
\end{align*}
\end{textblock*}

\begin{textblock*}{15cm}(0.5cm,6cm)
$\phi$ summarizes the noise dependence. Tuning the basal expression rates $P_{A}$ and $P_{B}$ is one way of generating ``cell types"
\end{textblock*}

\begin{textblock*}{15cm}(0.5cm,7.5cm)
This is useful when benchmarking algorithms which detect genuine biological variability or experimental variability
\end{textblock*}


\begin{textblock*}{3.5cm}(10cm,1.5cm)
\begin{figure}
\includegraphics[width=3.5cm]{master.png}
\end{figure}
\end{textblock*}

\end{frame}

\begin{frame}{Convergence tests for Markov chains}
\end{frame}

\begin{frame}{GRN Inference from simulated data}
\end{frame}

\begin{frame}{Autoencoding simulated data}
\end{frame}


\begin{frame}{Training on BBBC039 U2OS Cells}
\vspace{0.1in}
BBBC039: 200 images, 160 train + 40 validation, 256\;x\;256 random crop

\begin{center}
\includegraphics[width=0.8\textwidth]{weights.png}
\end{center}

We train a 3-channel semantic segmentation model with \textbf{weighted} cross-entropy loss:

\begin{equation*}
\mathcal{L} = \sum_{i,j} w_{ij}\log p_{ij}(\tilde{x}) = \sum_{i,j} w_{ij}\log \frac{\exp(-s_{ij}(\tilde{x}))}{\sum_{x\in\chi} \exp(-s_{ij}(\tilde{x}))}
\end{equation*}

$p_{ij}$ is the probability the model assigns a pixel to the true class $\tilde{x} \in \{\textrm{a}, \textrm{b}, \textrm{c}\}$

\end{frame}

\begin{frame}{Training on BBBC039 U2OS Cells}

\begin{center}
\includegraphics[width=0.85\textwidth]{input-train.png}
\includegraphics[width=0.85\textwidth]{target-train.png}
\includegraphics[width=0.85\textwidth]{output-train.png}
\end{center}

\end{frame}

\begin{frame}{Training on BBBC039 U2OS Cells}
Learning rate $\eta=0.01$, Batch-size $B=5$ (32 train iterations, 8 validation)
\begin{center}
\includegraphics[width=0.85\textwidth]{metrics.png}
\end{center}

\end{frame}



% Adding the option 'allowframebreaks' allows the contents of the slide to be expanded in more than one slide.
\begin{frame}[allowframebreaks]{References}
	\tiny\bibliography{references}
	\bibliographystyle{apalike}
\end{frame}

\end{document}
