% ---- ETD Document Class and Useful Packages ---- %
\documentclass{ucetd}
\usepackage{subfigure,epsfig,amsfonts}
\usepackage{natbib}
\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{amsthm}
\usepackage[toc,page]{appendix}
\usepackage[labelfont=bf]{caption}
\usepackage{rotating}
\usepackage[dvipsnames]{xcolor}
\usepackage{url}
 

%% Use these commands to set biographic information for the title page:
\title{Efficient Bayesian analysis of high dimensional biological systems with Markov Chain Monte Carlo}
\author{Clayton W. Seitz}
\department{Department of Physics}
\division{Physical Sciences}
\degree{Doctor of Philosophy}
\date{Spring 20XX}

%% Use these commands to set a dedication and epigraph text

\epigraph{Epigraph}

\begin{document}
%% Basic setup commands
% If you don't want a title page comment out the next line and uncomment the line after it:
\maketitle
%\omittitle

% These lines can be commented out to disable the copyright/dedication/epigraph pages
\makecopyright
%\makededication


%% Make the various tables of contents
\tableofcontents
%\listoffigures
%\listoftables

%\acknowledgments
% Enter Acknowledgements here

\abstract



\clearpage

\mainmatter

\chapter{Primer on exact Bayesian methods and variational inference}

\section{Markov Chain Monte Carlo}

\textcolor{red}{The invention of fast digital computers gives us the ability to simulate random processes and perform inference}

\subsection{Metropolis-Hastings and Gibbs sampling}

\subsection{Langevin Monte Carlo} 

\textcolor{red}{The metropolis adjusted Langevin algorithm (MALA) aka Langevin Monte Carlo}

\subsection{Hamiltonian Monte Carlo}

\subsection{Stochastic Gradient Langevin Dynamics}

\section{Variational Inference}

This won't be about deep learning, just introducing the concept so we can make a comparison with MCMC methods, in case that is relevant

\subsection{The evidence lower bound}


\chapter{A Bayesian approach for inferring neuronal connectivity from Ca2+ imaging data and Monte Carlo simulations}

\chapter{Bayesian inference of the kinetic parameters of interferon-gamma induced transcription}

\chapter{Stratifying immuogenic tumor substructure}

\textcolor{red}{I am interested in the relationship between tumor substructure i.e. heterogenity and how it relates to the degree of T-cell inflammation. We can be confident that certain malignant clusters are more immunogenic than others, which, to a first approximation, can be understood by measuring T-cell quantity in the sample. This \emph{could} then be supplemented by using T-cell signatures documented in the literature, differential expression, and clustering methods/mixture models to design fluorescent biomarkers. These will then be added to T-cell markers. In principle, this would give a more complete association of gene expression to the inflammaotry state, and would be a useful method in trying to understand changes during the evolution of cancer and following pharmacological treatment.}

\vspace{0.2in}

\textcolor{red}{For the implementation, I will use Seurat for preprocessing and visualization. Malignant cells are isolated first using standard methods e.g, filtering, normalization, and UMAP clustering}

\vspace{0.2in}

\textcolor{red}{This section will probably have to deviate significantly from the central theme of the dissertation, but it will be a major component of my work so it's necessary to include it}

%% Format a LaTeX bibliography

\makebibliography





\end{document}


