% Latex template: mahmoud.s.fahmy@students.kasralainy.edu.eg
% For more details: https://www.sharelatex.com/learn/Beamer

\documentclass[aspectratio=1610]{beamer}					% Document class

\setbeamertemplate{footline}[text line]{%
  \parbox{\linewidth}{\vspace*{-8pt}Establishing a quantitative framework for analyzing IFN$\gamma$ induction in HeLa cells \hfill\insertshortauthor\hfill\insertpagenumber}}
\setbeamertemplate{navigation symbols}{}

\usepackage[english]{babel}				% Set language
\usepackage[utf8x]{inputenc}			% Set encoding

\mode<presentation>						% Set options
{
  \usetheme{default}					% Set theme
  \usecolortheme{default} 				% Set colors
  \usefonttheme{default}  				% Set font theme
  \setbeamertemplate{caption}[numbered]	% Set caption to be numbered
}

% Uncomment this to have the outline at the beginning of each section highlighted.
%\AtBeginSection[]
%{
%  \begin{frame}{Outline}
%    \tableofcontents[currentsection]
%  \end{frame}
\usepackage{graphicx}					% For including figures
\usepackage{booktabs}					% For table rules
\usepackage{hyperref}	
\usepackage{tikz-network}				% For cross-referencing
\usepackage[absolute,overlay]{textpos}
\usepackage{bm}
\usepackage[font=small,labelfont=bf]{caption}				% For cross-referencing

\title{Establishing a quantitative framework for analyzing IFN$\gamma$ induction in HeLa cells}	% Presentation title
\author{Clayton W. Seitz}								% Presentation author
\date{\today}									% Today's date	

\begin{document}

% Title page
% This page includes the informations defined earlier including title, author/s, affiliation/s and the date
\begin{frame}
  \titlepage
\end{frame}


% The following is the most frequently used slide types in beamer
% The slide structure is as follows:
%
%\begin{frame}{<slide-title>}
%	<content>
%\end{frame}


\begin{frame}{Interferon-$\gamma$ induces differential gene expression in HeLa cells}

\vspace{0.1in}
Single cell transcriptome measurements of polyA mRNA for naÃ¯ve HeLa cells (N=90), induced with interferon gamma (50ng/mL) for 24h

\begin{figure}
\includegraphics[width=7cm]{volcano.png}
\end{figure}

{\tiny Siwek et al. 
\it{Activation of Clustered IFN$\gamma$ Target Genes Drives Cohesin-Controlled Transcriptional Memory} Cell 2020}

\end{frame}

\begin{frame}{Key questions}

\begin{itemize}
\item Does IFN$\gamma$ induce transcriptional bursts in HeLa cells?
\item Which genes?
\item What are the parameters of the burst (size, frequency, etc.)?
\item In general, it possible to correlate spatial patterning with transcriptional bursting, using only ensemble snapshots (FISH)?
\end{itemize}

\vspace{1in}

Need to establish analysis methods on a testing dataset... 


\end{frame}

\begin{frame}{Significant variability in STL1 mRNA counts per cell at 0.4M NaCl}

\begin{textblock*}{5cm}(3.25cm,0.5cm)
\begin{figure}
\includegraphics[width=10cm]{avg-count.png}
\end{figure}
\end{textblock*}

\vspace{7.25cm}
Error bars represent standard deviations from the mean\\
Cells marked ON for $>$ 3 STL1 mRNA in yeast

\end{frame}

\begin{frame}{Assessing STL1 mRNA count variability at the transcription site}

\begin{textblock*}{5cm}(3.25cm,0.5cm)
\begin{figure}
\includegraphics[width=10cm]{int-hist.png}
\end{figure}
\end{textblock*}

\vspace{7.25cm}
The median of the mRNA intensity distribution is used to determine the number of nascent RNA at the transcription site (TS)

\end{frame}

\begin{frame}{Assessing STL1 mRNA count variability at the transcription site}

\begin{textblock*}{5cm}(0.75cm,1.5cm)
\begin{itemize}
\item Brightest spot in the nucleus defined as putative TS
\item TS marked ACTIVE if $I>2*\mathrm{med}$
\item Nascent mRNA count is $\mathrm{round}(I/\mathrm{med})$
\item Count variability suggests asynchrony
\end{itemize}

\end{textblock*}

\begin{textblock*}{5cm}(7cm,0.75cm)
\begin{figure}
\includegraphics[width=8cm]{active-ts-dist-2.png}
\end{figure}
\end{textblock*}

\begin{textblock*}{5cm}(7cm,4.5cm)
\begin{figure}
\includegraphics[width=8cm]{active-ts-dist-1.png}
\end{figure}
\end{textblock*}

\end{frame}

\begin{frame}{Gene expression is stochastic and non-constitutive}


\begin{textblock*}{7cm}(1cm,1cm)
\begin{figure}
\includegraphics[width=7cm]{burst-1.png}
\end{figure}
\end{textblock*}

\begin{textblock*}{7cm}(8cm,1cm)
\begin{figure}
\includegraphics[width=6cm]{burst-2.png}
\end{figure}
\end{textblock*}


\begin{textblock*}{6cm}(1cm,5cm)
\hspace{0.5in}\textbf{Single-state models}
\begin{itemize}
\item RNAs are 'born' at a fixed rate
\item RNA counts are Poisson
\end{itemize}
\end{textblock*}

\begin{textblock*}{6cm}(8cm,5cm)
\hspace{0.5in}\textbf{Multi-state models}
\begin{itemize}
\item Promoter can be in multiple states (switching behavior)
\item RNA counts are not Poissonian
\end{itemize}

\end{textblock*}

\begin{textblock*}{15cm}(0.5cm,8cm)
Single-state models tend to \textcolor{red}{underestimate variance in RNA counts}
\end{textblock*}


\end{frame}

\begin{frame}{Gene expression is stochastic and non-constitutive (live-cell MS2-MCP)}
\begin{figure}
\includegraphics[width=12cm]{live-cell.png}
\end{figure}
{\tiny Forero-Quintero, et al. \textit{Live-cell imaging reveals the spatiotemporal organization of endogenous RNAPII phosphorylation at a single gene}. Nat Commun 2021}\\
\end{frame}

\begin{frame}{Ensemble averages and variances do not fully explain underlying transcription dynamics}

\begin{itemize}
\item Transcription is stochastic, meaning that RNA counts can only be understood in terms of a probability distribution
\vspace{0.2in}
\item High variance in mRNA counts suggests more complicated underlying dynamics which are not evident in ensemble averages
\vspace{0.2in}
\item We cannot assume that cells are bursting synchronously
\vspace{0.2in}
\item \textcolor{red}{Bursting phase has implications for correlating bursting with spatial organization via ensemble data (FISH)}
\vspace{0.1in}
\begin{itemize}
\item Classification of cells based on $P(X_{nuc},X_{cyto},X_{TS})$?
\item Can also look at the evolution of the spatial feature distributions
\end{itemize}
\end{itemize}

\end{frame}


\begin{frame}{A compartment model for IFN$\gamma$ induced gene expression}

Let $X$ represent an arbitrary RNA transcript of IFN$\gamma$ induced gene $G$. Assume two chromatin states (on and off)

\begin{align*}
\mathrm{Gene\;activation}: G_{off} &\overset{k_{on}}{\rightarrow} G_{on}\\
\mathrm{Gene\;inactivation}: G_{on} &\overset{k_{off}}{\rightarrow} G_{off}\\
\mathrm{Transcription}: G_{on} &\overset{k_{t}}{\rightarrow} G_{on} + X_{\mathrm{nuc}}\\
\mathrm{RNA \;Export}: X_{\mathrm{nuc}} &\overset{k_{exp}}{\rightarrow} X_{\mathrm{cyt}}\\
\mathrm{RNA\; degradation}: X_{\mathrm{cyt}} &\overset{\gamma}{\rightarrow} \emptyset\\
\end{align*}

Raw data collected post induction can be used to infer parameters

\begin{equation*}
\theta = \left( k_{on},k_{off},k_{t},k_{exp},\gamma\right)
\end{equation*}

\end{frame}

\begin{frame}{Bayesian parameter inference using ensemble snapshots}
\vspace{0.1in}

Likelihood-based methods can infer $\theta$ from ensemble snapshots (FISH data)

\begin{equation*}
\theta = \left( k_{on},k_{off},k_{t},k_{exp},\gamma\right)
\end{equation*}

One way is through maximum a posteriori estimation (MAP):

\begin{align*}
\theta^{*} = \underset{\theta}{\mathrm{argmax}} \;\;P(X|\theta)
\end{align*}

A more robust (but harder) way is via Bayesian inference, which lets us infer $\theta$ from $X$ while quantifying the uncertainty in our estimate:

\begin{align*}
P(\theta|\mathbf{x}) \propto P(\mathbf{x}|\theta)\pi(\theta) = \pi(\theta)\prod_{t} P(\mathbf{x}_{t}|\theta)
\end{align*}

The likelihood $P(X,t)$ is the solution to the chemical master equation at time $t$

\end{frame}

\begin{frame}{Kolmogorov's forward equation (chemical master equation)}

Dynamics on biochemical reaction networks are inherently stochastic and the state space is discrete. We can only write probabilities over the state space

\vspace{0.1in}

\begin{align*}
P(\mathbf{x}_{i},t) &= \sum_{j} T_{ji}(\mathbf{x}_{i},t|\mathbf{x}_{j},t-\Delta t)P(\mathbf{x}_{j},t-\Delta t)\\ 
&= \sum_{k} T_{k}(\mathbf{x}_{i},t|\mathbf{x}_{i}-\nu_{k},t-\Delta t)P(\mathbf{x}_{i}-\nu_{k},t-\Delta t)
\end{align*}

where $T_{k}$ is the probability of a reaction channel $k$ firing in the interval $(t,t+\Delta t)$.\\
\vspace{0.1in}
Taking the limit $\Delta t \rightarrow 0$ one can derive the forward Kolmogorov equation or chemical master equation (CME)

\begin{align*}
\frac{dP(\mathbf{x},t|\mathbf{x}_{0})}{dt} = \sum_{k} T_{k}(\mathbf{x}-\nu_{k})P(\mathbf{x}-\nu_{k},t) - T_{k}(\mathbf{x})P(\mathbf{x},t)
\end{align*}

\end{frame}

\end{document}