% Source: http://tex.stackexchange.com/a/5374/23931
\documentclass{article}
\usepackage[T1]{fontenc}
\usepackage[utf8]{inputenc}
\usepackage[margin=1in]{geometry}
\usepackage{amsmath}
\usepackage{graphicx}

\newcommand{\HRule}{\rule{\linewidth}{0.5mm}}
\newcommand{\Hrule}{\rule{\linewidth}{0.3mm}}

\makeatletter% since there's an at-sign (@) in the command name
\renewcommand{\@maketitle}{%
  \parindent=0pt% don't indent paragraphs in the title block
  \centering
  {\Large \bfseries\textsc{\@title}}
  \HRule\par%
  \textit{\@author \hfill \@date}
  \par
}
\makeatother% resets the meaning of the at-sign (@)

\title{Plan of Study}
\author{Clayton W. Seitz}
\date{\today}

\begin{document}
  \maketitle% prints the title block
\vspace{0.4in}

\begin{abstract}
\end{abstract}

\section{Introduction}

The intracellular environment harbors a delicate set of interactions between DNA, RNA, and protein in order to carry out cellular functions and perform signal processing. State of the art multi-omics technologies have fostered a generalization of the notion of a biochemical pathway to the large-scale biochemical network, in an attempt to understand the structure of these interactions across different cellular states. In the context of gene regulation, the biochemical network perspective has proven quite powerful, as it leverages the power of probabilistic graphical models thus revealing the conditional dependence structure of high-dimensional joint probability distribution over protein and or RNA copy numbers. Such an approach simultaneously permits clues towards unknown regulatory interactions or the impact of controlled perturbations on the dynamics of biochemical reaction networks. Recently, the inference of gene regulatory networks (GRNs) from static experimental data has become a relatively mature class of methods when applied to gene expression data from single-cell RNA-sequencing (Singh, 2018). Interactions between different genes are represented as a directed graph in which nodes represent genes and directed edges indicate a causual relationship between a source gene and destination gene. 

We have identified several shortcomings of this approach in need of development. First, there exists biological varability and technical noise in our datasets, introducing heterogeneity which is difficult to detect. This can be thought of as a byproduct of the curse of dimensionality, in which we are unable to directly infer the joint distribution over biomolecule counts. The inability to segregate samples either by discriminative or generative modeling may create the illusion that the joint distribution is unimodal - a kind of ``modal collapse''. Furthermore, by design, any network inference algorithm neglects biochemical processes when drawing directed edges between nodes. Inferring precise biochemical kinetics is a much more difficult problem, but remains desirable. Gene expression is thought to be inherently stochastic, governed by an unknown and highly complex coupled system of stochastic differential equations (SDEs). Our inability to assign further detail to network edges stems from our inability to determine the form of these differential equations and their parameterization, or even its stationary statistics. At any rate, Bayesian inference of more detailed kinetic parameters may be possible, under the assumption that transcriptional kinetics are solely governed by a Hill function of transcription factor concentration (Burton, 2021). 

Of course, there are also a number of other confounding factors, such as the spatial organization of key biomolecules, the biophysical details of transcription factor binding, DNA accessibility, and RNA preprocessing, which enter into this hypothetical system of equations. We choose to focus our attention on spatial organization, as this information can be readily measured in multiplexed RNA-imaging experiments. Spatial regulation could contribute to non-Hill dynamics.  Network inference is frequently carried out on RNA-seq datasets or multiplexed RNA-FISH while neglecting the role of the placement of proteins in space and time. Transcription factors mediate the causual effect between the expression of a source gene and a target gene, making this information valuable in resolving the nature of this interaction. Finally, substantial evidence has surfaced that genes are not expressed constitutively; rather, gene expression occurs in a bursting fashion (Dar 2012; Larrson 2019; Tunnacliffe 2020) which has been suggested to occur due to switching of a gene promoter between inactivate and activate transcriptional states. Bursting phenomena challenge our assumptions of stationarity and ergodicity of the joint distribution over gene expression, posing important theoretical questions.

In this study, we use the drug-treatment response of WM989 melanoma cells to treatment with a chemotherapy drug Vemurafenib - a BRAF inhibitor. Statistical treatment has revealed a core set of genes and their logical interactions, which are thought to be of importance during the development of resistance to chemotherapy treatment \emph{in-vitro} (Shaffer 2017). Development of resistance to treatment has been proposed to a transient state of gene expression which has been attributed to non-genetic expression variability from transcriptional bursting (Schuh 2020). However, the mechanism by which drug treatment alters gene expression irreversibly remains unclear. We expect that this question can begin to be answered by probing alterations in the logical structure of the gene regulatory network alongside Bayesian inference of parameters of the functions governing transcription rates.

\subsection{Identifying key genomic regulators in melanoma}

\subsection{Transcriptional bursting: a source of non-genetic variability}

\subsection{Spatial properties of gene expression}

\section{Methods}

\subsection{Theoretical Methods}

\subsubsection{Statistical mechanics of transcription factor binding}

\subsubsection{The Gillespie algorithm}

\subsubsection{Stability analysis of stochastic differential equations}

\subsubsection{Inferring biochemical networks from data}

\subsubsection{Learning interaction functions with Gaussian Processes}

\subsection{Experimental Methods}

\subsubsection{Multiplexed fluorescence in-situ hybridization (FISH)}

\subsubsection{Techniques for high-throughput image processing}


\end{document}